pkg/ingress/group_loader.go

```diff
@@ -260,22 +260,17 @@ func (m *defaultGroupLoader) sortGroupMembers(members []ClassifiedIngress) ([]Cl
 	}
 
 	groupMemberWithOrderList := make([]groupMemberWithOrder, 0, len(members))
-	explicitOrders := sets.NewInt64()
 	for _, member := range members {
 		var order = defaultGroupOrder
 		exists, err := m.annotationParser.ParseInt64Annotation(annotations.IngressSuffixGroupOrder, &order, member.Ing.Annotations)
 		if err != nil {
 			return nil, errors.Wrapf(err, "failed to load Ingress group order for ingress: %v", k8s.NamespacedName(member.Ing))
 		}
 		if exists {
-			if order < minGroupOrder || order > maxGroupOder {
+			if order <= minGroupOrder || order >= maxGroupOder {
```

Comment:
lets make the range inclusive i.e: [1000, -1000]
Otherwise, the change will be backwards incompatible